testsuite/features/secondary/srv_distro_cobbler.feature

```diff
@@ -59,11 +59,6 @@ Feature: Cobbler and distribution autoinstallation
     Then I should see a "Autoinstallation: fedora_kickstart_profile" text
     And I should see a "Autoinstallation Details" link
 
-  Scenario: Autoinstallation profiles page
```

Comment:
Why don't we need this anymore?

Reply:
Because in fact, we never need it.
It does nothing.
First it navigate to autoinstall overview, to then jump to profiles, to then check for the text "Distributions" which it's not even on the page but in the left menu, together with Overview and Profiles.